amethyst_network/src/network_socket.rs

```diff
@@ -135,14 +132,21 @@ where
                     // Get the event
                     match deserialize_event::<E>(packet.payload()) {
                         Ok(event) => {
+                            let mut connection_exists = false;
+
                             // Get the NetConnection from the source
                             for connection in (&mut net_connections).join() {
                                 if connection.target_addr == packet.addr() {
+                                    connection_exists = true;
                                     connection
                                         .receive_buffer
                                         .single_write(NetEvent::Packet(event.clone()));
                                 };
                             }
+
+                            if !connection_exists {
+                                panic!("No connection exitsts");
```

Comment:
Might not want to do that.

Reply:
was testing code, thanks for pointing out!